### PR TITLE
send signup vs login url based on auth selection

### PR DESCRIPTION
**Pull request by xavluiz**
bug: https://www.notion.so/softwaredotcom/Email-login-9e19a97ce9d547bba594889c40ee37b1

email signup should use the /email-signup view and the login selection should use the /onboarding view

https://www.loom.com/share/c97c0040191b4b27b48fbb9b4e519e84

### CODE REVIEW

File: src/menu/AccountManager.ts [hunk 1]
```diff
@@ -55,14 +55,14 @@ async function accountMenuSelection(placeholderItem: any) {
...
+  showAuthMenuOptions(LOGIN_LABEL, false /*isSignup*/);
 }
 
 function showSignUpMenuOptions() {
-  showAuthMenuOptions(SIGN_UP_LABEL);
+  showAuthMenuOptions(SIGN_UP_LABEL, true /*isSignup*/);
 }
 
-function showAuthMenuOptions(authText) {
+function showAuthMenuOptions(authText, isSignup = true) {
```
Comment (mmclead) [inline src/menu/AccountManager.ts:65]: is it a named param or not?  we have both being used in this file

Comment (xavluiz) [reply src/menu/AccountManager.ts]: I added the types in my last commit update and a loom video. sorry, i'm not sure which param you're referring to? the isSignup one?

**Review by bjacobson26 — APPROVED** [review]
lgtm - i would remove the vsix from this pr since this will be > 2.4.2

**Comment by xavluiz** [comment]
> lgtm - i would remove the vsix from this pr since this will be > 2.4.2

will do. thanks

**Pull request merged**